Automattic/newspack-theme
# Standardizing font sizing and styles

### All Submissions:


### Changes proposed in this Pull Request:

Though it looks like a lot, this PR largely lines up the font variables in the theme with how they're sized in the designs, and makes some minor tweaks to font styles (like colours and weights).

### How to test the changes in this Pull Request:

1. Apply the PR.
2. Basically make sure nothing looks broken; it doesn't need a thorough review, as additional work is needed to make sure the correct font sizes are applied to the correct locations. 

### Other information:
